mne/io/nihon/nihon.py

```diff
@@ -272,10 +274,19 @@ def _read_nihon_annotations(fname):
             n_logs = np.fromfile(fid, np.uint8, 1)[0]
             fid.seek(t_blk_address + 0x14)
             t_logs = np.fromfile(fid, '|S45', n_logs)
-            for t_log in t_logs:
+            if version == _valid_headers[-1]:
```

Comment:
The check for `if version == _valid_headers[-1]` happens a few times. It would be better not to rely on `_valid_headers` having a specific order, so I suggest a convenience variable, maybe like this

```py
is_version_1200A = version == 'EEG-1200A V01.00'  # add around line 265 or so
```

Not sure if it's the best naming choice since I'm unfamiliar with what this specific header is used for (a specific MEG device model?)

then this (and the other if-lines) would become 
```suggestion
            if is_version_1200A:
```